### ISSUE TITLE
Reverted PR remains visible in "Local Pull Request Branches" tab of sidebar

### ISSUE THREAD

**Issue by curtdrummondvpc** [microsoft/vscode-pull-request-github#4855]
- Extension version:  v0.65.2023051809
- VSCode Version: v1.78.2
- OS: Mac OSX 13.3.1 (a)

Steps to Reproduce:

Unclear how to reproduce this, but there's a PR that was reverted a long time ago that remains visible in the "Local Pull Request Branches" section of the extension's sidebar.
1.  Open VSCode
2.  Open Github Pull Requests and Issues extension sidebar
3. Pull request is visible even though it was closed and then reverted.

See screenshot for the PR that won't go away, as well as a section of settings.json where I have tried to follow other advice found (e.g. https://github.com/microsoft/vscode-pull-request-github/issues/4809) to try to ignore this PR.

![Screenshot 2023-05-25 at 1 29 15 PM](https://github.com/microsoft/vscode-pull-request-github/assets/99760397/f239161a-d64c-470a-bc0f-1405b58b1b0f)

**Comment by BrotherErryn** [comment]
I'm having this same issue with one that was merged. It has no outstanding reviews/etc. and I have verified that it is indeed merged. It was a release merge into main from development, if that makes any difference. Have attempted the same settings @curtdrummondvpc indicated above, but no change.

I could ignore this pretty easily if it didn't pop up a notice reminding me it was there every time I switched back into the development branch.
